File: Programmers/JAVA/lv1/lv1_부족한 금액 계산하기.java
https://programmers.co.kr/learn/courses/30/lessons/82612

class Solution {
    public long solution(int price, int money, int count) {
        long answer = -1;
        long sum = 0;
        
        for(int i = 1; i <= count; i++){
            sum += price*i;
        }
        answer = sum - money;
        
        if(answer <= 0)
            answer = 0;
        
        return answer;
    }
}

File: Programmers/C#/lv1/lv1_부족한 금액 계산하기.cs
https://school.programmers.co.kr/learn/courses/30/lessons/82612?language=csharp

class Solution
{
    public long solution(int price, int money, int count)
    {
        long sum = 0;

        while (count > 0)
        {
            sum += price * (count--);
        }

        return (sum > money) ? sum - money : 0;
    }
}